mne/time_frequency/tfr.py

```diff
@@ -1349,6 +1457,14 @@ def tfr_multitaper(inst, freqs, n_cycles, time_bandwidth=4.0,
     .. versionadded:: 0.9.0
     """
 
+    if not isinstance(time_range, tuple):
+        raise TypeError('time_range should be a tuple.')
+    elif len(time_range) != 2:
+        raise ValueError('time_range should be of size 2.')
+    tmin, tmax = time_range
+    if not isinstance(tmin, (int, type(None))) \
+       or not isinstance(tmax, (int, type(None))):
+        raise TypeError("time_range elements should be integers or None.")
```

Comment:
This will fail for e.g. `float` and `ndarray` inputs though... probably better to use the existing function if possible.